frontend/src/forms/RegisterForm.jsx

```diff
@@ -58,7 +62,16 @@ export default function RegisterForm() {
             required: true,
             message: 'Enter Password Again!',
           },
+          ({ getFieldValue }) => ({
+            validator(_, value) {
+              if (!value || getFieldValue('password') === value) {
+                return Promise.resolve();
+              }
+              return Promise.reject(new Error('The new password that you entered do not match!'));
```

Comment:
change it to this new Error('The two passwords that you entered do not match!')

Reply:
made the requested changes